src/cosalib/cmdlib.py

```diff
@@ -240,3 +240,16 @@ def get_timestamp(entry):
     # Older versions only had ostree-timestamp
     ts = j.get('coreos-assembler.build-timestamp') or j['ostree-timestamp']
     return parse_date_string(ts)
+
+
+def string_truth(val):
```

Comment:
I'm not a big fan of these sorts of ad-hoc "convert a string to true/false".   It seems much cleaner to simply have the empty string `""` be "no validation", and anything else be a path to a file.